frontend/ui/widget/titlebar.lua

```diff
@@ -48,6 +48,7 @@ local TitleBar = OverlapGroup:extend{
     left_icon_size_ratio = 0.6,
     left_icon_rotation_angle = 0,
     left_icon_tap_callback = function() end,
+    -- set hold_callback=false to not handle the event and let it propagate; otherwise the event is discarded
```

Comment:
May be move this below, after all the icon stuff, as it's valid for any callback (would work with tap too I think):
```diff
    right_icon_hold_callback = function() end,
    right_icon_allow_flash = true,
+        -- set any of these _callback to false to not handle the event
+        -- and let it propagate; otherwise the event is discarded
```